### ISSUE TITLE
Unable to use qt-mountainview

### ISSUE THREAD

**Issue by yukidai** [flatironinstitute/mountainlab-js#73]
Hi, I'm trying to visualize my data using qt-mountainview but it seems like that the ml-prv-create function fails to create prv files on my computer. Please take a look at it. Thank you very much!

After tying in:

>qt-mountainview --raw=raw.mda.prv --filt=filt.mda --pre=pre.mda --firings=firings_raw.mda --cluster_metrics=metrics_tagged.json --samplerate=30000

It returned:

> GLib-GIO-Message: 10:54:41.034: Using the 'memory' GSettings backend.  Your settings will not be saved or shared with other applications.
> ** qt-mountainview ; origin: /home/tjd/anaconda3/conda-bld/git_cache/github.com/flatironinstitute/qt-mountainview ; commit: heads/master-0-gff77bac
> ** Compiled using Qt version: 5.6.2 (/home/user/conda/envs/mlab on host: 'rime'
> Setting up object registry...
> Parsing command-line parameters...
> Creating MVContext...
> Setting up context...
> Creating prv object for: filt.mda
> Creating prv object for: pre.mda
> Creating prv object for: firings_raw.mda
> Setting up main window...
> Adding controls to main window...
> Opening initial views...
> Starting event loop...
> [ Getting processor spec... ]
> [ Checking inputs and substituting prvs ... ]
> [ Computing process signature ... ]
> /tmp/EE11_01_20180604.nt1.mountain/firings_raw.mda
> (node:17436) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'original_checksum' of undefined
>     at /home/user/conda/envs/mlab/lib/node_modules/mountainlab/mlproc/prv_utils.js:203:24
>     at /home/user/conda/envs/mlab/lib/node_modules/mountainlab/mlproc/prv_utils.js:192:7
> (node:17436) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
> (node:17436) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> ":::: Failed to open DiskReadMda32 file: /tmp/mountainlab-tmp/tmp_long_term/41796105db66a8dced781f086f21cd6db0a8be22_templates_out.tmp"
> "File does not exist: /tmp/mountainlab-tmp/tmp_long_term/41796105db66a8dced781f086f21cd6db0a8be22_templates_out.tmp"
> Unable to read chunk of templates in cluster detail view
> 

I also tried to use the ml-prv-create function directly and it returned error messages:

> 
> Creating prv record for file ... : pre.mda
> (node:16338) UnhandledPromiseRejectionWarning: TypeError: console.err is not a function
>     at /home/user/conda/envs/mlab/lib/node_modules/mountainlab/mlproc/prv_utils.js:72:15
>     at /home/user/conda/envs/mlab/lib/node_modules/mountainlab/mlproc/prv_utils.js:192:7
> (node:16338) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
> (node:16338) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by alexmorley** [comment]
Hi there! Could you please post the output for `ml-prv-locate raw.mda.prv`? If it finds a file it would be work to check that file exists.

I think there may be a bug here as well but it could be a day or two before someone has a chance to fix.

**Comment by yukidai** [comment]
I tried with 

> ml-prv-locate raw.mda.prv

And it returned with the correct location of my data:

> 
/media/user/DATA/Projects/eeyd_a5GABA_Project/EE11_FF+_Experiment/EE11/01_20180604/EE11_01_20180604.mda/EE11_01_20180604.nt1.mda

**Comment by alexmorley** [comment]
Thanks ... could you also post the exact command you tried with `ml-prv-create`?

**Comment by yukidai** [comment]
Yes, its just:

> (mlab) user:EE11_01_20180604.nt1.mountain$ ml-prv-create pre.mda

**Comment by alexmorley** [comment]
Weird! OK thanks I will try and look into this properly this week.

**Comment by theilmbh** [comment]
Hi all,
We had a similar error on a fresh install of mountainlab.  We found that the `sumit.json` and `process_cache.json` in `~/.mountainlab/database` were completely empty on a fresh install, and this seemed to be the cause of the problem.  We simply added `[]` to these empty files and then things would run.  I just wanted to bring this to your attention in case anyone else runs in to similar problems.
